src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -152,63 +116,25 @@ internal void Unregister()
                     m_solutionEventsCookie = VSCOOKIE_NIL;
                 }
             }
-
-            // Unregister this object from IVsRunningDocTableEvents events
-            if (m_runningDocTableEventsCookie != VSCOOKIE_NIL)
-            {
-                IVsRunningDocumentTable runningDocTable = ServiceProvider.GlobalProvider.GetService(typeof(SVsRunningDocumentTable)) as IVsRunningDocumentTable;
-                if (runningDocTable != null)
-                {
-                    runningDocTable.UnadviseRunningDocTableEvents(m_runningDocTableEventsCookie);
-                    m_runningDocTableEventsCookie = VSCOOKIE_NIL;
-                }
-            }
         }
+        #region IVsUpdateSolutionEvents2
```

Comment:
>IVsUpdateSolutionEvents2 [](start = 16, length = 24)

As a note, we are implementing this entire interface just to remove some temporary files. I'm looking for an alternative to that.